docs/tutorials/getting_started.ipynb

```diff
@@ -31,1089 +31,160 @@
     "\n",
```

Comment:
<p>if box length is a quantity, then we wouldn't have to manually make all these things quantities for sending to the plot function!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/21cmfast/tuesday/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>